src/es/components/atoms/breadcrumb/Breadcrumb.js

```diff
@@ -50,39 +55,39 @@ export default class Breadcrumb extends Shadow() {
       :host > ul > li > a {
-        color: var(--a-color, var(--color-secondary, var(--color, pink)));
+        color: var(--a-color, var(--color-secondary, ${color})); /* use variable "color" */
         font-weight: var(--a-font-weight, var(--font-weight, normal));
         text-align: var(--a-text-align, unset);
-        text-decoration: var(--a-text-decoration, var(--text-decoration, none));
+        text-decoration: var(--a-text-decoration, ${textDecoration}); /* use variable "textDecoration" */
         text-underline-offset: var(--a-text-underline-offset, unset);
         display: var(--a-display, inline);
         margin: var(--a-margin, var(--content-spacing, unset)) auto;
       }
       :host a:hover, :host a:active, :host a:focus {
         color: var(--a-color-hover, var(--color-hover-secondary, var(--color-hover, var(--color, green))));
-        text-decoration: var(--a-text-decoration-hover, var(--text-decoration-hover, var(--a-text-decoration, var(--text-decoration, none))));
+        text-decoration: var(--a-text-decoration-hover, var(--text-decoration-hover, var(--a-text-decoration, ${textDecoration}))); /* use variable "textDecoration" */
       }
       :host > ul > li > a {
-        color: var(--a-color, var(--color-secondary, var(--color, pink)));
-        text-decoration: var(--a-text-decoration, var(--text-decoration, none));
+        color: var(--a-color, var(--color-secondary, ${color})); /* use variable "color" */
+        text-decoration: var(--a-text-decoration, ${textDecoration}); /* use variable "textDecoration" */
         text-underline-offset: var(--a-text-underline-offset, unset);
       }
       :host > ul > li > a:hover, :host > ul > li > a:active, :host > ul > li > a:focus {
         color: var(--a-color-hover, var(--color-hover-secondary, var(--color-hover, var(--color, green))));
-        text-decoration: var(--a-text-decoration-hover, var(--text-decoration-hover, var(--a-text-decoration, var(--text-decoration, none))));
+        text-decoration: var(--a-text-decoration-hover, var(--text-decoration-hover, var(--a-text-decoration, ${textDecoration}))); /* use variable "textDecoration" */
       }
       :host > ul > li > span {
-        ${this.getAttribute('icon') !== 'false' ? `background-image: var(--background-image, url(${this.getAttribute('icon') || '_import-meta-url_../web-components/src/icons/chevron_right.svg'}));` : ''}
+        ${icon !== 'false' ? `background-image: var(--background-image, url(${icon || '_import-meta-url_../web-components/src/icons/chevron_right.svg'}));` : ''}
         background-position: center;
         background-repeat: no-repeat;
         background-size: 1em;
-        ${this.getAttribute('icon') !== 'false' ? 'color: transparent;' : ''}
+        ${icon !== 'false' ? 'color: transparent;' : ''}
         min-width: 1em;
       }
       @media only screen and (max-width: _max-width_) {
         :host {
-          margin: var(--margin-mobile, var(--margin, 0)) !important;
```

Comment:
this margin and width with fallback 0 is questionable and possible not needed. needs to be testet... if needed for some reason leave !important to overseed the width and margin being set by o-body to all children within main. Also !important is not as ugly within a webcomponent as it used to be with a global only stylesheet.